install/0000_30_machine-api-operator_10_service.yaml

```diff
@@ -18,3 +18,32 @@ spec:
   selector:
     k8s-app: machine-api-operator
   sessionAffinity: None
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: machine-api-controllers
+  namespace: openshift-machine-api
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: machine-api-controllers-tls
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+  labels:
+    k8s-app: controller
+spec:
+  type: ClusterIP
+  ports:
+  - name: machine-mtrc
+    targetPort: machine-mtrc
+    port: 8441
+  - name: machineset-mtrc
+    targetPort: machineset-mtrc
+    port: 8442
+  - name: nodelink-mtrc
```

Comment:
Do we think is there any value we can get out of exposing metrics for the nodelink controller?
I'd expect us to eventually drop it and embed that knowledge into the machine controller.

Reply:
I think that dropping those metrics and nodelink removal could be done at the same time WDYT?

Reply:
I can't see any value on conflating that here. If you agree there's no much value on exposing the nodelink metrics let's just undo the changes that expose the controller so we alleviate the operational complexity introduced by this PR.
Dropping the nodelink controller is a completely different discussion that does not need to happen in this PR.